appium/server-side/python/run-tests_android.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+##
+## Work in progress! The dependency installations need to be done to the
+## container so that we don't need to install them here.
+##
+TEST=${TEST:="BitbarSampleAppTest.py"} #Name of the test file
+
+##### Cloud testrun dependencies start
+echo "Extracting tests.zip..."
+unzip tests.zip
+
+echo "Installing pip for python"
+curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+sudo python get-pip.py
+
+echo "Installing Appium Python Client 0.24 and xmlrunner 1.7.7"
+chmod 0755 requirements.txt
+sudo pip install -r requirements.txt
+
+echo "Starting Appium ..."
+
+/opt/appium/appium/bin/appium.js >appium.log 2>&1 &
```

Comment:
the appium should probably be started with the new wrapper command.

Android: appium-1.6 --log-no-colors --log-timestamp
iOS: appium-1.6 -U ${UDID} --log-no-colors --log-timestamp --show-ios-log